### ISSUE TITLE
Snippet issue on other connected clients

### ISSUE THREAD

**Issue by engbmsdev** [pusher/atom-pair#15]
When I use a code snippet (e.g. `script` + `TAB` on HTML page), my friend receives the evaluated snippet + snippet identifier.
### My workstation

![screen shot 2015-02-24 at 14 27 01](https://cloud.githubusercontent.com/assets/7578169/6350159/acde3692-bc31-11e4-80bf-88c639526178.png)
![screen shot 2015-02-24 at 14 27 08](https://cloud.githubusercontent.com/assets/7578169/6350165/b580d23c-bc31-11e4-9f0c-d46afa9a33d9.png)
### My friend

![myfriend](https://cloud.githubusercontent.com/assets/7578169/6350209/0be69da0-bc32-11e4-92b9-6ba7fc08849d.png)

Thank you!

**Comment by jpatel531** [comment]
Ah thanks for bringing this to our attention! I'll get on it right away :)

**Comment by snollygolly** [comment]
I looked into this last night and was curious how you intend to fix it.  It looks like that's a `snippet:expand` event, could we bind to that and somehow tell the client that the text we're replacing is part of a snippet replace?

**Comment by jpatel531** [comment]
Hi there! I realized I was completely short-sighted in how I defined a 'buffer substitution' -- it used to be when the old range contained the new range, not realizing it can actually be the other way round. E.g when expanding snippets, the new range is a superset of the old range. 

A bit mad how simple the solution was. I'll roll this out in version 2.0.0, which has a bunch of new features, e.g. multiple tab sharing
